packages/browser-repl/config/karma.conf.js

```diff
@@ -0,0 +1,35 @@
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```

Comment:
ooooo whatchu thinking of doing with puppeteer? :D

Reply:
run tests I hope :D